iam-login-service/src/main/java/it/infn/mw/iam/config/DataSourceConfig.java

```diff
@@ -0,0 +1,50 @@
+package it.infn.mw.iam.config;
```

Comment:
This is my usual fault :) Forgetting the license header! you can copy it from other Java classes or from LICENSE file. I know we should upgrade it. We'll do it in one shot for all the files soon.

Reply:
I have added the header, thanks for pointing this out.